samples/bookinfo/src/productpage/requirements.txt

```diff
@@ -27,5 +27,5 @@ thrift==0.11.0
 tornado==4.5.3
 urllib3==1.26.5
 visitor==0.1.3
-Werkzeug==0.15.5
+Werkzeug==2.1.1
```

Comment:
Would you verify that things work to build as doc'd here: https://github.com/istio/istio/tree/master/samples/bookinfo

Reply:
after upgrading Flask, it can be used normally

Reply:
update the framework of Werkzeug, I have tested it and submitted it to another pr @ericvn @howardjohn